lib/middleware/resolve-project-version-updates.js

```diff
@@ -7,7 +7,7 @@ module.exports = settings => {
     const model = get(req.body, 'model');
     const action = get(req.body, 'action');
     if (model === 'projectVersion' && (action === 'update' || action === 'patch')) {
-      return resolver({ data: req.body, update: noop })
+      return resolver({ data: req.body, update: noop, patch: noop })
```

Comment:
This is currently failing here: https://github.com/UKHomeOffice/asl-workflow/blob/master/lib/hooks/resolve/index.js#L13

Passing a no-op makes it not fail.